src/MediaCollections/Models/Media.php

```diff
@@ -40,7 +40,7 @@ class Media extends Model implements Responsable, Htmlable
 
     protected $guarded = [];
 
-    protected $casts = [
+    protected array $casts = [
```

Comment:
as explained here you are not allowed to cast this property this way: https://stackoverflow.com/a/61302038

if you need to cast it for eg. php stan you could add a php doc block
```
    /**
     * @var array<string, string|class-string<\Illuminate\Contracts\Database\Eloquent\Castable>>
     */
     protected $casts = [
```